plugins/modules/database/misc/redis.py

```diff
@@ -5,6 +5,7 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import absolute_import, division, print_function
+import re
```

Comment:
Better add this to the imports below `DOCUMENTATION` etc. The top should only have the `from __future__` import and `__metaclass__ = type`.